src/app/messages.ts

```diff
@@ -38,12 +60,39 @@ export const errMessages = {
     }
   },
   Login: {
-    
+    "login": "Oops... Invalid email or password, please type it again."
+  },
+  PasswordValidation: {
+    mismatch: {
+      "mismatch": "Password mismatch ..."
```

Comment:
Same as further down with registration mismatch
"The passwords you have entered do not match each other. Please enter the same password."